internals/secrethub/account_email_verify.go

```diff
@@ -23,7 +23,10 @@ func NewAccountEmailVerifyCommand(io ui.IO, newClient newClientFunc) *AccountEma
 
 // Register registers the command, arguments and flags on the provided Registerer.
 func (cmd *AccountEmailVerifyCommand) Register(r command.Registerer) {
-	clause := r.Command("verify-email", "Send an email to the registered email address to prove you own that email address.")
+	clause := r.Command("verify-email", "Resend verification email to the registered email address.")
+	clause.HelpLong("When you create your account, a verification email is automatically sent to the email address you used to sign up.\n" +
+		"In case anything goes wrong (e.g. the email ended up in your junk folder), this command lets you resend the verification email.\n" +
```

Comment:
Same here. I'm fine with leaving the newlines out alltogether, as it's a fairly short piece of text, but if you want to add them, I'd say add double newlines, otherwise it will look weird with certain screen dimensions.